### PR TITLE
Composer: update dev dependencies

### PR DESCRIPTION
**Pull request by Roman3349**
Signed-off-by: Roman Ondráček <ondracek.roman@centrum.cz>

### CODE REVIEW

**Comment by f3l1x** [comment]
It requires to work on it Roman. ;-) Could you please fix it? @Roman3349

File: phpstan.neon [hunk 1]
```diff
@@ -7,6 +8,4 @@ parameters:
 		# Ignore PHP soft-failing functions
 		- message: '#.+(int|string|string\>)\|false.+#'
 		  path: %currentWorkingDirectory%/src/SchemaDefinition/Entity/EntityAdapter.php
-
-		- message: '#Cannot call method allowsNull\(\) on ReflectionType\|null.#'
-		  path: %currentWorkingDirectory%/src/SchemaDefinition/Entity/EntityAdapter.php
+	reportUnmatchedIgnoredErrors: false
```
Comment (mabar) [inline phpstan.neon]: Please remove unmatched ignored errors instead

File: src/SchemaDefinition/Entity/EntityAdapter.php [hunk 2]
```diff
@@ -148,7 +149,10 @@ protected function getProperties(string $type): array
 	private function getPropertyType(ReflectionProperty $property): ?string
 	{
 		if (PHP_VERSION_ID >= 70400 && ($type = Reflection::getPropertyType($property) !== null)) {
-			return ($property->getType()->allowsNull() ? '?' : '') . $type;
+			$propertyType = $property->getType();
+			if ($propertyType !== null) {
+				return ($propertyType->allowsNull() ? '?' : '') . $type;
+			}
```
Comment (mabar) [inline src/SchemaDefinition/Entity/EntityAdapter.php]: You've ran into `unsupported or uncovertible variable type "1"`, right?

**Comment by f3l1x** [comment]
@Roman3349 Hey, could you rebase it? Thx.

**Comment by Roman3349** [comment]
@f3l1x Done

**Pull request merged**